core/templates/base-components/header.template.html

```diff
@@ -10,6 +10,7 @@
 <meta property="og:site_name" content="Oppia">
 <meta property="og:description" content="<%= htmlWebpackPlugin.options.meta.description %>">
 
+<link rel="apple-touch-icon" href="/assets/images/logo/favicon.webp">
 <link rel="apple-touch-icon" href="/assets/images/logo/favicon.png">
```

Comment:
Do we need both?

Reply:
Yes when webp is not supported as you mentioned safari doesn't support webp we can provide png

Reply:
But then we don't need the `webp` version here since from some source it seems that `apple-touch-icon` only supports PNG.

Reply:
@darksun27 ping.

Reply:
done!